distros/ubuntu1604/rules

```diff
@@ -25,7 +25,10 @@ override_dh_auto_configure:
             -DZM_SOCKDIR="/var/run/zm"                 \
             -DZM_TMPDIR="/tmp/zm"                      \
             -DZM_CGIDIR="/usr/lib/zoneminder/cgi-bin"  \
-            -DZM_CONTENTDIR="/var/cache/zoneminder"
+            -DZM_CONTENTDIR="/var/cache/zoneminder"    \
```

Comment:
I might have gotten ahead of myself on this one.... when I modify zoneminder and our packaging scripts to not do symlinks, I expect to need this to create the folder. If not, I'll remove it.